docs/design/proposals/userID-mapping.md

```diff
@@ -0,0 +1,66 @@
+# UserID Mapping
+
+CephCSI mounts RBD or CephFS volumes on nodes using specific ceph userID. Currently,
+there is no mechanism to track and expose this nodeID:userID mapping information per volume.
+
+## Problem
+
```

Comment:
Please mention why _RBD watchers_ (or `sysfs` inspection) and _CephFS sessions_ are not a good approach.

Reply:
Added a section "alternatives" mentioning other approaches and its drawbacks.